modules/custom/az_global_footer/az_global_footer.install

```diff
@@ -77,3 +77,71 @@ function az_global_footer_update_1020701() {
 
   return t('Updated %count Campus Safety menu link(s) in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update Phonebook links
+ *
+ * Phonebook links in the global footer will be 
+ * updated with new primary domain.
+ */
+function az_global_footer_update_1020706() {
+  $updated_count = 0;
+  $menu_link_content_ids = \Drupal::entityQuery('menu_link_content')
+    ->accessCheck(FALSE)
+    ->condition('menu_name', 'az-footer-resources')
+    ->condition('title', 'Phonebook')
+    ->condition('link__uri', 'https://directory.arizona.edu/phonebook')
+    ->execute();
+
+  foreach ($menu_link_content_ids as $id) {
+    /**
+     * @var \Drupal\menu_link_content\Entity\MenuLinkContent $menu_link_content
+     */
+    $menu_link_content = MenuLinkContent::load($id);
+    if ($menu_link_content !== NULL) {
+      // phpcs:disable
+      /* @phpstan-ignore-next-line */
+      $menu_link_content->link->uri = 'https://phonebook.arizona.edu';
+      // phpcs:enable
+      $menu_link_content->save();
+      $updated_count++;
+    }
+  }
+
+  return t('Updated %count Phonebook menu link(s) in the global footer.', ['%count' => $updated_count]);
+}
+
+/**
+ * Update Directory links
+ *
+ * Directory links in the global footer will be 
+ * updated with new primary domain.
+ */
+function az_global_footer_update_1020706() {
+  $updated_count = 0;
+  $menu_link_content_ids = \Drupal::entityQuery('menu_link_content')
+    ->accessCheck(FALSE)
+    ->condition('menu_name', 'az-footer-resources')
+    ->condition('title', 'A-Z Index')
+    ->condition('link__uri', 'https://directory.arizona.edu/index')
+    ->execute();
+
+  foreach ($menu_link_content_ids as $id) {
+    /**
+     * @var \Drupal\menu_link_content\Entity\MenuLinkContent $menu_link_content
+     */
+    $menu_link_content = MenuLinkContent::load($id);
+    if ($menu_link_content !== NULL) {
+      // phpcs:disable
+      /* @phpstan-ignore-next-line */
+      $menu_link_content->link->title = 'Directory';
```

Comment:
```suggestion
      $menu_link_content->link->title = 'Directory';
```
I'm unsure if this aspect will work. I'm looking to update the URL and the link title.

Reply:
This did not work. URLs were updated correctly, but not the link title.